File: online-judge-solutions/Codeforces/1606A.py
#https://codeforces.com/problemset/problem/1606/A
#Lang		:	Python 3.8
#Time		:	31 ms
#Memory         :	0 KB
n=int(input())
for x in range(n):
    s=input()
    print(s[:-1]+s[0])

File: online-judge-solutions/Codeforces/1606A.cpp
//https://codeforces.com/problemset/problem/1606/A
//Lang		:	GNU C++14
//Time		:	0 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int n;
	cin>>n;
	while(n--){
		string s;cin>>s;
		cout<<s.substr(0,s.size()-1)+s[0]<<endl;
	}
	return 0;
}
